src/services/authentication/authentication.service.js

```diff
@@ -0,0 +1,4 @@
+import * as firebase from "firebase";
```

Comment:
If you are on firebase 9 please checkout master for a more up to date reference:
https://github.com/mobinni/MealsToGo/blob/master/src/services/authentication/authentication.service.js